src/PresentationalComponents/Dashboard/Dashboard.js

```diff
@@ -100,7 +102,7 @@ const Dashboard = ({ workloads }) => {
                                 }
                             </Suspense>
                             <Suspense fallback={ <Loading /> }>
-                                {permission.subscriptions &&
+                                {!isFedramp && permission.subscriptions &&
```

Comment:
Can we just kill the whole suspense component? seems kinda odd to load for ah thing we wont be showing

Reply:
<3